src/doc/src/reference/environment-variables.md

```diff
@@ -37,13 +37,15 @@ system:
   Useful to set up a build cache tool such as `sccache`. See
   [`build.rustc-wrapper`] to set via config. Setting this to the empty string
   overwrites the config and resets cargo to not use a wrapper.
-* `RUSTC_WORKSPACE_WRAPPER` --- Instead of simply running `rustc`, for workspace
-  members Cargo will execute this specified wrapper, passing
-  as its command-line arguments the rustc invocation, with the first argument
-  being the path to the actual rustc. It affects the filename hash
-  so that artifacts produced by the wrapper are cached separately.
-  See [`build.rustc-workspace-wrapper`] to set via config. Setting this to the empty string
-  overwrites the config and resets cargo to not use a wrapper for workspace members.
+* `RUSTC_WORKSPACE_WRAPPER` --- Instead of simply running `rustc`, for workspace members Cargo will
```

Comment:
I wonder if we could push people toward reading `build.rustc-workspace-wrapper`, instead of duplicating the doc.

Reply:
Is there a way to link to the other docs?

Reply:
Never mind. I found it is not a doc for `BUILD_RUSTC_WORKSPACE_WRAPPER`, so probably fine with duplicates.